### PR TITLE
Build from spec file types

### PR DESCRIPTION
**Pull request by kimsauce**
### Description

see https://saucedev.atlassian.net/browse/TCD-2239

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/api-testing/build-from-spec.md [hunk 1]
```diff
@@ -2,25 +2,22 @@
 id: build-from-spec
 title: Building an API Test from a Spec File
 sidebar_label: Build Test from Spec File
-description: "The available spec files you can automatically generate from are: Swagger, RAML, OpenAPI 3, API Blueprint, I/O Docs, SOAP WSDL, and Postman Collection."
+description: "Automatically generate tests from a spec file in one of the following formats: Swagger, OpenAPI 3, and Postman Collection."
```
Comment (LoryApiFortress) [inline docs/api-testing/build-from-spec.md:5]: @kimsauce Automatically generate tests from a spec file in one of the following formats: OpenAPI 3, HAR file and Postman Collection (btw, the only spec file is OpenAPI 3, the other files are not really spec files)

File: docs/api-testing/build-from-spec.md [hunk 2]
```diff
@@ -2,25 +2,22 @@
...
 
 ## What You'll Need
 
 * A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up))
 * An existing API Testing Project. For details on how to create one, see [API Testing Quickstart](/api-testing/quickstart/)
-* A REST API Specification file in one of our supported formats: <Highlight color="#37b732">Swagger</Highlight> <Highlight color="#4e5b2e">OpenAPI 3</Highlight> <Highlight color="#ff6c37">Postman Collection</Highlight>
+* A REST API Specification file in one of our supported formats:
+   * OpenAPI 3.0 or higher
+   * Postman Collection
+   * Swagger
```
Comment (LoryApiFortress) [inline docs/api-testing/build-from-spec.md:19]: @kimsauce remove Swagger

File: docs/api-testing/build-from-spec.md (hunk 1 above)
Comment (kimsauce) [reply docs/api-testing/build-from-spec.md]: Good point!

File: docs/api-testing/build-from-spec.md [hunk 3]
```diff
@@ -2,25 +2,18 @@
...
-
-Using the API Testing **HTTP Client**, you can generate tests from external endpoints or files such as an OpenAPI Specification (spec) file.
+Using the API Testing **HTTP Client**, you can generate tests from external endpoints or files such as an OpenAPI Specification (spec).
 
 ## What You'll Need
 
 * A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up))
 * An existing API Testing Project. For details on how to create one, see [API Testing Quickstart](/api-testing/quickstart/)
-* A REST API Specification file in one of our supported formats: <Highlight color="#37b732">Swagger</Highlight> <Highlight color="#4e5b2e">OpenAPI 3</Highlight> <Highlight color="#ff6c37">Postman Collection</Highlight>
+* An OpenAPI Specification file (v3.0 or higher) or HAR (HTTP Archive) file
```
Comment (LoryApiFortress) [inline docs/api-testing/build-from-spec.md:16]: @kimsauce even though Postman is not a "spec" file and there's a different page for it, I think we should add it in the list and link the specific page

**Review by LoryApiFortress — COMMENTED** [review]
@kimsauce please update the image for "import button" (in both files) b/c now it also contains the label "har"

**Review by LoryApiFortress — APPROVED** [review]

**Pull request merged**